app/grandchallenge/core/models.py

```diff
@@ -62,16 +65,33 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     def status_to_string(self):
+
         status = (
             f"Your request to join {self.object_name}, "
             f"sent {self.format_date(self.created)}"
         )
         if self.status == self.PENDING:
-            status += ", is awaiting review"
+            try:
+                user_is_verified = self.user.verification.is_verified
+            except ObjectDoesNotExist:
+                user_is_verified = False
+
+            if (
+                not user_is_verified
+                and self.base_object.access_request_handling
+                == utils.access_requests.AccessRequestHandlingOptions.ACCEPT_VERIFIED_USERS
+            ):
+                status = (
+                    "You need to verify your account in order to get access"
+                )
```

Comment:
This status isn't necessarily correct. People _could_ still be accepted, despite not being verified. 

```suggestion
                status += ", is awaiting review. Your request will be automatically accepted if you verify your account."
```